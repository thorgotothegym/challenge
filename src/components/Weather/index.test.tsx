import { render } from "@testing-library/react";
import { Weather } from "./";

describe("Weather", () => {
  it("Renders Component According To Snapshot", () => {
    const { container } = render(<Weather />);

    expect(container).toMatchSnapshot();
  });
});
