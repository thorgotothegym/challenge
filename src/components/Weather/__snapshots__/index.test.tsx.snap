// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather Renders Component According To Snapshot 1`] = `
<div>
  <div
    data-testid="weatherID"
  >
    <h1 />
    <p>
       Celcius
    </p>
    <img
      alt="Icon"
    />
  </div>
</div>
`;

exports[`Weather component id should be found data-testid 1`] = `
<div>
  <div
    data-testid="weatherID"
  >
    <h1 />
    <p>
       Celcius
    </p>
    <img
      alt="Icon"
    />
  </div>
</div>
`;
